### PR TITLE
consumer: Refactor the code to handle F28 Two week atomic

### PR DESCRIPTION
**Pull request by sayanchowdhury**
Signed-off-by: Sayan Chowdhury <sayan.chowdhury2012@gmail.com>

### CODE REVIEW

File: fedimg/consumers.py [hunk 1]
```diff
@@ -99,19 +98,19 @@ def consume(self, msg):
...
+
+            if (cloud_meta and atomic_meta) is None:
+                images_meta = cloud_meta or atomic_meta
             else:
-                images_meta.extend(get_value_from_dict(
-                    compose_metadata, 'images', 'payload',
-                    'images', 'AtomicHost', 'x86_64'))
+                images_meta = cloud_meta + atomic_meta
 
-        if images_meta is None:
```
Comment (dustymabe) [inline fedimg/consumers.py]: The changes here are mostly cosmetic right? There is no bug getting fixed here?

Comment (sayanchowdhury) [reply fedimg/consumers.py]: Yes, these changes are cosmetic changes and fixes the broken tests.

File: fedimg/consumers.py [hunk 2]
```diff
@@ -99,19 +98,19 @@ def consume(self, msg):
...
+                compose_metadata, 'images', 'payload',
+                'images', 'AtomicHost', 'x86_64')
+
+            if (cloud_meta and atomic_meta) is None:
+                images_meta = cloud_meta or atomic_meta
             else:
-                images_meta.extend(get_value_from_dict(
-                    compose_metadata, 'images', 'payload',
-                    'images', 'AtomicHost', 'x86_64'))
+                images_meta = cloud_meta + atomic_meta
```
Comment (dustymabe) [inline fedimg/consumers.py]: this if/else block is confusing.. I think I'd rather see something like:
```
images_meta = []
if cloud_meta:
    images_meta.extend(cloud_meta)
if atomic_meta:
    images_meta.extend(atomic_meta)
```

File: fedimg/consumers.py (hunk 1 above)
Comment (dustymabe) [reply fedimg/consumers.py]: Since the changes don't actually change any functionality, how do they fix broken tests?

**Comment by dustymabe** [comment]
LGTM. if tests pass

**Pull request merged**